torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -498,6 +498,17 @@ RegisterOperators reg({
           };
         }),
 
+    Operator(
+        "prim::IgnoredPythonOp(...) -> ()",
+        [](const Node* node) -> Operation {
+          return [](Stack& stack) {
+            throw JITException(
+                "This Python function annotated to be ignored"
+                " and cannot be executed");
```

Comment:
How about this: "... and hasn't been included in the exported binary, meaning that it can't be executed now. Make sure that ignored operations are never executed after import."